game/loca/ru_ru/common.php

```diff
@@ -12,8 +12,8 @@
 
 $LOCA["ru"]["PLANET_HOME"] = "Главная планета";
 $LOCA["ru"]["PLANET_COLONY"] = "Колония";
-$LOCA["ru"]["PLANET_PHANTOM"] = "Planet";
-$LOCA["ru"]["PLANET_ABANDONED"] = "Покинутая колония";
+$LOCA["ru"]["PLANET_PHANTOM"] = "Планета";
+$LOCA["ru"]["PLANET_ABANDONED"] = "Покинутая планета";
```

Comment:
In the original, after colonizing the 10th planet in the galaxy, there was a bug known as “overlib”. I fixed it and intentionally made the name “колония” instead of “планета” to avoid confusion with “Уничтоженная планета” (deleted by the player).
Wouldn't want to change this translation for that reason :)